src/searchers/usual.js

```diff
@@ -0,0 +1,73 @@
+import {playSound, BEEP, WRAP} from '../sound';
+import {poscmp} from '../utils';
+
+export default class {
+  constructor(blocks) {
+    this.blocks = blocks;
+  }
+
+  getLabel() {
+    return 'Search by usual';
+  }
+
+  find(forward, e) {
```

Comment:
It looks like your plan is to have a different JS file for each search type. Personally I don't love the idea (why not just put them all in the same file?), but my bigger concern is duplicate code. Won't they ALL have a `find` function, which works exactly the same way? `initSearch` is also going to be nearly identical. The only thing different will be how `this.searchMatches` is constructed.

Reply:
You are right. The plan right now is to use inheritance to enable the reuse.